src/style.css

```diff
@@ -0,0 +1,26 @@
+* {
+  margin: 0;
+}
```

Comment:
don't reset margins for all elements
better to specify some elements you need to reset

Reply:
- Reset browser's default `margin`    I rad this in the task